modules/products/confluence/variables.tf

```diff
@@ -99,4 +99,32 @@ variable "pvc_claim_name" {
 variable "enable_synchrony" {
   description = "If true, Collaborative editing service will be enabled."
   type        = bool
+}
+
+variable "db_snapshot_identifier" {
+  description = "Snapshot identifier for RDS."
+  type        = string
+  default     = null
+}
+
+variable "db_master_username" {
+  description = "Master username for the RDS instance."
+  type        = string
+  default     = null
+}
+
+variable "db_master_password" {
+  description = "Master password for the RDS instance."
+  type        = string
+  default     = null
+}
+
+variable "db_snapshot_build_number" {
+  description = "Confluence build number of the database snapshot."
+  type        = string
+  default     = null
+  validation {
+    condition     = var.db_snapshot_build_number == null || can(regex("^[0-9]{4}$", var.db_snapshot_build_number))
```

Comment:
Is the build number sharp 4 digits? If it can be 3 or 5 digits (builds in before or future) then better use something like `^([0-9]).{2,4}$`

Reply:
It's 4 digits at least from Confluence 7.13 (minimum supported version of Helm chart). 
https://developer.atlassian.com/server/confluence/confluence-build-information/ 
And I don't see it bumping to 5 digits in the near future, so will keep it as 4-digit for now. But will make the validation message more explicit as you mentioned below.

Reply:
If the documentation says at least 4 digit so perhaps we are safe to add this validation:
`^([0-9]).{3,4}$` to cover 4 digits and 5 digits for future and copy the validation message from confluence documentation.